src/Auth0.ManagementApi/Clients/HooksClient.cs

```diff
@@ -0,0 +1,108 @@
+        {
+            return Connection.SendAsync<Hook>(HttpMethod.Post, BuildUri("hooks"), request, DefaultHeaders);
+        }
+
+        /// <summary>
+        /// Deletes a hook.
+        /// </summary>
+        /// <param name="id">The ID of the hook to delete.</param>
+        /// <returns>A <see cref="Task"/> that represents the asynchronous delete operation.</returns>
+        public Task DeleteAsync(string id)
+        {
+            return Connection.SendAsync<object>(HttpMethod.Delete, BuildUri($"hooks/{EncodePath(id)}"), null, DefaultHeaders);
+        }
+
+        /// <summary>
+        /// Retrieves a list of all hooks.
+        /// </summary>
+        /// <param name="request">Specifies criteria to use when querying hooks.</param>
+        /// <param name="pagination">Specifies pagination info to use when requesting paged results.</param>
+        /// <returns>An <see cref="IPagedList{T}"/> containing the hooks requested.</returns>
+        public Task<IPagedList<Hook>> GetAllAsync(GetHooksRequest request, PaginationInfo pagination)
+        {
+            if (request == null)
+                throw new ArgumentNullException(nameof(request));
+            if (pagination == null)
+                throw new ArgumentNullException(nameof(pagination));
+
+            return Connection.GetAsync<IPagedList<Hook>>(BuildUri("hooks",
+                new Dictionary<string, string>
+                {
+                    {"enabled", request.Enabled?.ToString().ToLower()},
+                    {"fields", request.Fields},
+                    {"include_fields", request.IncludeFields?.ToString().ToLower()},
+                    {"page", pagination.PageNo.ToString()},
+                    {"per_page", pagination.PerPage.ToString()},
+                    {"include_totals", pagination.IncludeTotals.ToString().ToLower()}
+                }), DefaultHeaders, hooksConverters);
+        }
+
+        /// <summary>
+        /// Retrieves a hook by its ID.
+        /// </summary>
+        /// <param name="id">The ID of the hook to retrieve.</param>
+        /// <param name="fields">
+        /// A comma separated list of fields to include or exclude (depending on
+        /// <paramref name="includeFields" />) from the result, empty to retrieve all fields.
+        /// </param>
+        /// <param name="includeFields">
+        /// True if the fields specified are to be included in the result, false otherwise (defaults to
+        /// true).
+        /// </param>
+        /// <returns>The <see cref="Hook" /> that was requested.</returns>
+        public Task<Hook> GetAsync(string id, string fields = null, bool includeFields = true)
+        {
+            return Connection.GetAsync<Hook>(BuildUri($"hooks/{EncodePath(id)}",
+                new Dictionary<string, string>
+                {
+                    {"fields", fields},
+                    {"include_fields", includeFields.ToString().ToLower()}
```

Comment:
The GetHooks API endpoint does not support sending `include_fields`: https://auth0.com/docs/api/management/v2#!/Hooks/get_hooks_by_id

Reply:
Removed this now